src/main/_docs/ide/ide-git-svn.md

```diff
@@ -184,14 +191,84 @@ Files in project explorer and editor tabs can be colored according to their Git
-With the diff window it is possible to review changes between two states of code. To view the diff, use `Git`->`Compare`->`<Select-to-what>` from main menu. If more than one file has changed the list of changed files will be opened first. To select a file to compare double-click on it or select a file then click the `Compare` button.
+## Committing changes
+To commit your changes use the git commit window. To show it click `Git` -> `Commit...` from main menu or just use `Alt + c` shortcut.
+
+![git-commit-tree-view.png]({{base}}{{site.links["git-commit-tree-view.png"]}}){:style="width: auto"}
+
+You can select files which will be added to index and committed.
+
+All files in the selected package / folder in the project explorer will be checked by default. After selecting files, type the commit message in the text area. Optionally you could select `Amend previous commit` to replace previous commit (for more details [see git docs](https://git-scm.com/docs/git-commit#git-commit---amend)).
+
+Push the commit to a remote repository by checking `Push committed changes to` checkbox and selecting branch to push into. 
+
+Click `Commit` to proceed (the `Commit` button is active as long as at least 1 file is selected and a commit message is present or `Amend previous commit` is checked).
+
+Behavior for files in the list view is the same as in compare window (see [Reviewing changed files](#git-compare) section). Double clicking a file will open diff window with it.
+
+## Pushing changes to remote repository
+To push your commits to remote use the git push dialog.
+To open it use `Git` -> `Remotes...` -> `Push` from main menu or `Alt + Shift + c` shortcut.
+
+![git-push.png]({{base}}{{site.links["git-push.png"]}}){:style="width: auto"}
+
+Then choose remote repository to push into and select local and remote branch.
+Optionally by checking `Force push` is possible to push with force.
+After this, hit `Push` button to proceed.
+
+## Pulling changes from remote repository
+To get changes from remote repository use the git pull dialog.
+To open it use `Git` -> `Remotes...` -> `Pull` from main menu or `Alt + p` shortcut.
+
+![git-pull.png]({{base}}{{site.links["git-pull.png"]}}){:style="width: auto"}
+
+Then choose remote repository to pull from and select remote and local branch.
+Optionally by checking `Rebase instead of merge` is possible to keep your local commits on top 
+(for more information [see git docs](https://git-scm.com/docs/git-pull#git-pull--r)).
+After this, hit `Pull` button to proceed.
+
+## Managing branches
+To manage your git branches use branches window.
+To open it use `Git` -> `Branches...` from main menu or `Ctrl + b` shortcut.
+
+![git-branches-window.png]({{base}}{{site.links["git-branches-window.png"]}}){:style="width: auto"}
+
+In the middle of the window is placed a list of all branches in current project.
+It is possible to display only local or only remote branches by `Show branches` drop-down above.
```

Comment:
Please note that it is also possible to display both remote and local in one list

Reply:
Yes, this is default behaviour. Here is said, that a user can filter branches in this way